### ISSUE TITLE
Addition of multi languages in "Accessibility "

### ISSUE THREAD

**Issue by Valdnet** [nextcloud/server#14607]
I suggest adding multi languages to "Accessibility". This is missing for the entire Nextcloud translation.

**Comment by Valdnet** [comment]
In "Accessibility" there are no words to translate in Transifex:
`Themes`
`Fonts`

![obraz](https://user-images.githubusercontent.com/47037905/54752352-0e860d80-4bde-11e9-8923-9a40f66b6b4a.png)

**Comment by rullzer** [comment]
@juliushaertl please have a look

**Comment by juliushaertl** [comment]
Fixed on 16.

**Comment by Valdnet** [comment]
None to be translated into Transifex:

- Universal access is very important to us. We follow web standards and check to make everything usable also without mouse, and assistive software such as screenreaders. We aim to be compliant with the (...) 2.1 on AA level, with the high contrast theme even on AAA level.

- If you find any issues, don’t hesitate to report them on (...). And if you want to get involved, come join (...)!

**Comment by juliushaertl** [comment]
Ah sorry, I just checked Nextcloud 16 and it seems that for 15 the translation of accessibility is indeed missing.

**Comment by Valdnet** [comment]
This description is still missing to be translated into Transifex:

![obraz](https://user-images.githubusercontent.com/47037905/65706442-c95dff00-e08a-11e9-9795-bf17bb4594f4.png)

App:
Nextcloud 17.0.0 RC2
Accessibility 1.3.0

**Comment by kesselb** [comment]
cc @juliushaertl @rullzer

**Comment by juliushaertl** [comment]
The issue is that our [translationtool](https://github.com/nextcloud/docker-ci/blob/master/translations/translationtool/src/translationtool.php#L330-L331) does not work properly with the multiline t-call using template strings https://github.com/nextcloud/server/blob/81613d22469bc20faad4ec342832f53f541cb469/apps/accessibility/src/Accessibility.vue#L57-L64

**Comment by Valdnet** [comment]
@juliushaertl Still this line of code:
https://github.com/nextcloud/server/blob/81613d22469bc20faad4ec342832f53f541cb469/apps/accessibility/src/Accessibility.vue#L71-L75

**Comment by juliushaertl** [comment]
@Valdnet Not sure what you mean exactly. The line you mention is also missing in the transifex sync but https://github.com/nextcloud/docker-ci/pull/172 should fix it once merged.

**Comment by Valdnet** [comment]
> Not sure what you mean exactly.

That's exactly what I meant for it to appear in Trasifex :wink:

**Comment by TP75** [comment]
Thanks.  However, let's hope we receive this in NC 16 also.
